File: codeforces/85.die_roll_[9A]/die_roll.cpp
#include <iostream>
#include <algorithm>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int y, w;
  unsigned short int n, d;
  unsigned short int i;
  cin >> y >> w;
  d = 6;
  n = d - max(y, w) + 1;
  i = 2;
  while (i <= n) {
    if ((n % i) == 0 && (d % i) == 0) {
      n = n / i;
      d = d / i;
    }
    else {
      i++;
    }
  }
  cout << n << "/" << d << endl;
  return 0;
}

File: codeforces/low_difficulty/800_points/85.die_roll_[9A]/die_roll.py
# Author: José Rodolfo (jric2002)
y, w = list(map(int, input().split(" ")))
d = 6
n = d - max(y, w) + 1
i = 2
while (i <= n):
  if ((n % i) == 0 and (d % i) == 0):
    n = int(n / i)
    d = int(d / i)
  else:
    i += 1
print(n, d, sep="/")